app/views/catalog/_document.html.erb

```diff
@@ -0,0 +1,8 @@
+<% # container for a single doc -%>
+<article class="document <%= render_document_class document %> document-position-<%= document_counter%> " data-document-counter="<%= document_counter %>" itemscope itemtype="<%= document.itemtype %>">
+  <div class="row">
+  <%= render_document_partials document,
+    blacklight_config.view_config(document_index_view_type).partials,
+    document_counter: document_counter %>
+  </div>
+</article>
```

Comment:
We could move the availability code here (after the `</div>` and before `</article>`:

```
  <div class="blacklight-availability">
    <% unless document.hathi_links && document.hathi_links[:etas_item] && Settings&.hide_etas_holdings -%>
        <%= render partial: 'index_availability', locals: { document: document } unless Settings&.readonly %>
    <% end -%>
  </div>
  ```

Reply:
@nicktiberi would you mind giving this a try as well?